helpers/build_packages.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# droid-hal device packaging converter: from monolithic to modular
+# Copyright (c) 2015 Jolla Ltd.
+# Contact: Simonas Leleiva <simonas.leleiva@jollamobile.com>
```

Comment:
Also wrong. Here should go the proper licence preamble of dhd (BSD-3-Clause) and authors are Alin and Oleid (attribute that this code has been referenced from sfa-mer, if WTFPL exercises attribution)